OSPSuite.Presentation.Importer/Core/IDataSourceFile.cs

```diff
@@ -0,0 +1,48 @@
+﻿using OSPSuite.Infrastructure.Import.Services;
+using System.Collections.Generic;
+
+namespace OSPSuite.Presentation.Importer.Core
+{
+   /// <summary>
+   /// Single file containing the data, e.g. excel file or csv file
+   /// </summary>
+   public interface IDataSourceFile
+   {
+      Dictionary<string, IDataSheet> DataSheets { get; }
+	   string Path { get; set; }
+
+      IDataFormat Format { get; set; }
+
+      IList<IDataFormat> AvailableFormats { get; set; }
```

Comment:
why set?

Reply:
is being set from the outside in IImporter (the service)